tests/test/scheduler/test_executor.cpp

```diff
@@ -330,6 +330,34 @@ TEST_CASE_METHOD(TestExecutorFixture,
     REQUIRE(restoreCount == 0);
 }
 
+TEST_CASE_METHOD(TestExecutorFixture,
+                 "Test executing function repeatedly and flushing",
+                 "[executor]")
+{
+    std::shared_ptr<BatchExecuteRequest> req =
+      faabric::util::batchExecFactory("dummy", "simple", 1);
+    uint32_t msgId = req->messages().at(0).id();
+    std::vector<std::string> actualHosts;
+
+    int numRepeats = 20;
+    for (int i = 0; i < numRepeats; i++) {
+        std::vector<std::string> actualHosts =
+          executeWithTestExecutor(req, false);
```

Comment:
This `actualHosts` variable isn't used is it? It's also shadowing the `actualHosts` variable in the outer scope, so I don't see what either are doing.